tools/lsp/ui/components/widgets/color-widget.slint

```diff
@@ -22,15 +21,17 @@ export component ColorWidget inherits GridLayout {
 
     callback test-color-binding(text: string) -> bool;
     callback set-color-binding(text: string);
+    callback update-floating-editor();
 
-    spacing-vertical: EditorSpaceSettings.default-spacing;
+    padding-left: 16px;
```

Comment:
I prefer the old way of doing this with the `GridLayout` with a `ChildIndicator` in the first column. I did it like that so that you can change the `ChildIndicator` as you see fit and the widgets will continue to work... with this approach you hard-code that the `ChildIndicator` has a width of 16px.

This is not a blocker, I just wanted to explain my reasoning.